src/main/java/fungeye/cloud/domain/enities/Grow.java

```diff
@@ -81,4 +86,20 @@ public Box getBox() {
     public void setBox(Box box) {
         this.box = box;
     }
+
+    public Mushroom getMushroom() {
+        return mushroom;
+    }
+
+    public void setMushroom(Mushroom mushroom) {
+        this.mushroom = mushroom;
+    }
+
+    public Set<Harvest> getHarvests() {
```

Comment:
Glad to see this is a collection of harvests as many grows involve multiple harvests in what are called flushes